kani-compiler/src/codegen_boogie/compiler_interface.rs

```diff
@@ -170,7 +170,7 @@ impl BoogieCodegenBackend {
         if !tcx.sess.opts.unstable_opts.no_codegen && tcx.sess.opts.output_types.should_codegen() {
             let mut pb = boogie_file.to_path_buf();
             pb.set_extension("bpl");
-            debug!("Writing Boogie file to {}", pb.display());
+            println!("Writing Boogie file to {}", pb.display());
```

Comment:
It does make sense as an `info!`, but it seems we're currently not enabling the `Level::INFO` messages? (I tried switching to `info!`, and nothing got printed).

Eventually, this should only be a debug message. But for now, since one has to call the Boogie verifier manually, it is convenient to print the name of the Boogie file so that one can copy/paste it in the command.

Reply:
There are two ways to change the compiler log verbosity, the first one is to pass `--debug` to Kani. The second one is via `KANI_LOG` environment variable. The nice thing about the second is that you can control verbosity per module.

https://model-checking.github.io/kani/cheat-sheets.html#debug

Reply:
Cool. I changed it to `info!`.